moveit_ros/warehouse/warehouse/src/initialize_demo_db.cpp

```diff
@@ -109,6 +109,10 @@ int main(int argc, char** argv)
   for (std::size_t i = 0; i < gnames.size(); ++i)
   {
     const robot_model::JointModelGroup* jmg = psm.getRobotModel()->getJointModelGroup(gnames[i]);
+    if (jmg == nullptr)
+    {
```

Comment:
Is this parenthesis required?

Reply:
There is no need to modify because it may or may not be present. The reason is as described in the comment of PR # 22.

Reply:
Sorry,There is no similar writing in the source.I will fix it.

Reply:
OK